Makefile

```diff
@@ -1,3 +1,8 @@
 update_metadata:
 	(cd libPhoneNumberTests && ./metadataGenerator) && \
-		(cd libPhoneNumber && ./GeneratePhoneNumberHeader.sh)
+		(cd libPhoneNumber && ./GeneratePhoneNumberHeader.sh) && \
+		(cp libPhoneNumber/NBGeneratedPhoneNumberMetaData.h MetaDataPlistCreator) && \
+		(cd MetaDataPlistCreator && MetaDataPlistCreator) && \
+		(cp MetaDataPlistCreator/phoneNumberMap.plist libPhoneNumber) &&\
+		(rm MetaDataPlistCreator/phoneNumberMap.plist) &&\
```

Comment:
This is just `mv` :-)

Reply:
no, if the target exists the move will fail, the cp not (IIRC).

Reply:
`mv` overwrites by default, both `cp` and `mv` have a `-i` flag that says "check first".